### PR TITLE
loads course metadata from sanity

### PR DESCRIPTION
**Pull request by joelhooks**

### CODE REVIEW

File: src/lib/playlists.ts [hunk 1]
```diff
@@ -283,6 +285,7 @@ export async function loadPlaylist(slug: string, token?: string) {
   })
 
   const {playlist} = await graphQLClient.request(query, variables)
+  const courseMeta = await loadCourse(playlist.id)
 
-  return playlist
+  return {...playlist, ...courseMeta}
```
Comment (joelhooks) [inline src/lib/playlists.ts:290]: idea: just merge these together, eventually the idea of a "course" is a combination of a finite set of resource types. egghead backend focuses on media primarily, other aspects are migrated to sanity

**Comment by vojtaholik** [comment]
I noticed that instructor wasn't showing up below course title so I queried rest of the required data and passed them appropriately: aa9166f

File: src/lib/courses.ts [hunk 2]
```diff
@@ -1,179 +1,80 @@
...
+    url
+  },
+	'illustrator': collaborators[]->[role == 'illustrator'][0]{
+    'name': person->name,
+    'image': person->image.url
+  },
+	'instructor': collaborators[]->[role == 'instructor'][0]{
+    'name': person->name,
+    'avatar_url': person->image.url,
+    'url': person->slug.current
```
Comment (theianjones) [inline src/lib/courses.ts:40]: I noticed that this url is `/instructors/:instructor-slug` where it should be `/q/resources-by-:instructor-slug

File: src/components/pages/courses/instructor-profile.tsx [hunk 3]
```diff
@@ -23,7 +23,7 @@ const InstructorProfile: React.FunctionComponent<{
         <h4 className="text-gray-700 dark:text-gray-400 text-sm leading-tighter">
           Instructor
         </h4>
-        <Link href={`/instructors/${url}`}>
+        <Link href={`/q/resources-by-${url}`}>
```
Comment (theianjones) [inline src/components/pages/courses/instructor-profile.tsx:26]: I realized that it was a hard coded url 👀

**Review by theianjones — APPROVED** [review]
LGTM

**Pull request merged**